internal/backends/python/grab.go

```diff
@@ -298,12 +299,38 @@ func filterImports(ctx context.Context, foundPkgs map[string]bool) (map[string][
 
 	pkgs := map[string][]api.PkgName{}
 
+	moduleRoots := []string{"."}
```

Comment:
`src` would be in `From`, unfortunately as the comment I left in fc5eb624ed13232da4721adb8ae471b889825f9b mentions, we don't have a way to recognize that `src/module1/app.py` shouldn't necessarily find `src/module2/lib.py`, so we just check to see whether we have _any_ local modules that match the first package segment.

I can remove the stuff that maps `From`, but since it's part of the spec I'm inclined to keep it since it's closer to where we want to be eventually. otoh, yagni may be more appropriate, I could really go either way here.

Reply:
Okay! I'm fine with leaving it how you have it.